openui5/jquery.sap.d.ts

```diff
@@ -0,0 +1,187 @@
+// Type definitions for OpenUI5
```

Comment:
Looks like the latest stable release is `1.40`. So write that as `// Type definitions for OpenUI5 1.40`.